web-ng/src/app/services/auth/auth.service.ts

```diff
@@ -74,4 +76,21 @@ export class AuthService {
     await this.afAuth.signOut();
     return this.router.navigate(['/']);
   }
+
+  private async getUserEmail() {
+    const user = await this.user$.pipe(take(1)).toPromise();
```

Comment:
Since the UI is not reactive (i.e., it only checks the user permissions once when created)m would to make this code more readable can we instead keep the `currentUser` as a member of this class? We could subscribe to `user$` in the constructor, for example: `this.user$.subscribe(user => this.currentUser = user)`.

Then all of the auth methods can be sync instead of async.